lib/core-classes/AssetService.ts

```diff
@@ -42,23 +39,18 @@ export class AssetService {
         }
       }
     };
-
```

Comment:
Why did you removed all the single blank lines here? I feel that when the code is separated in "paragraphs", it's easier to read, what do you think?

Reply:
When I need to clean the code for easier reading, I split it in more little function or I add comment. 
Keep blank line in a function is not a reflex that I have, but I understand the idea.

Reply:
Usually comments inside functions should be avoided because the naming choices should be self-explanatory.  
I do agree for the small function splitting but here the blank lines where dividing code blocks or instructions, for example
![image](https://user-images.githubusercontent.com/4447392/166941204-c8d9dfae-3833-4677-9bd5-0aee12ecd801.png)

What bother you with extra blank lines?

Reply:
The two blank line given in example was deleted two quickly, sorry :s
No particular problem with extra blank line (on one of my other job, the rule was "never extra blank line" and that have perhaps format me to delete all of them. )